scout/server/blueprints/panels/views.py

```diff
@@ -153,8 +153,16 @@ def panel(panel_id):
 def panel_update(panel_id):
     """Update panel to a new version."""
     panel_obj = store.panel(panel_id)
-    update_version = request.form.get("version", None)
-    new_panel_id = store.apply_pending(panel_obj, update_version)
+    if request.form.get("cancel_pending"):
+        updated_panel = store.reset_pending(panel_obj)
+        if updated_panel.get("pending") is None:
+            flash("Pending actions were correctly canceled!", "success")
+        elif updated_panel is None:
```

Comment:
If `updated_panel` is `None` the first check will give an error. So these two needs to change place. Perhaps add a test ti that scenario as well?

Reply:
Thanks I'll fix it. I'd like to add some other fixes to this PR as well!

Reply:
I've fixed it. I think I'll leave this PR as it is and open another PR to fix more panel-related issues..  Could you check again @moonso? Thanks!